packages/storage/src/reference.ts

```diff
@@ -118,7 +120,36 @@ export class Reference {
 }
 
 /**
- * Uploads a blob to this object's location.
+ * Uploads data to this object's location.
+ * The upload is not resumable.
+ * @public
+ * @param ref - Storage Reference where data should be uploaded.
+ * @param data - The data to upload.
+ * @param metadata - Metadata for the newly uploaded string.
+ * @returns An UploadTaskNonResumableSnapshot
+ */
+export async function uploadBytes(
+  ref: Reference,
+  data: Blob | Uint8Array | ArrayBuffer,
+  metadata: Metadata | null = null
```

Comment:
the signature should be `metadata?: Metadata` according to the API proposal. It applies to `uploadBytesResumable` and `uploadString` as well.

Reply:
Done.